static/templates/bookend.hbs

```diff
@@ -1,8 +1,21 @@
 {{! Client-side Mustache template for rendering the trailing bookend.}}
-
-{{#if bookend_content}}
 <div class="{{#if trailing}}trailing_bookend{{/if}} bookend sub-unsub-message">
-    <span>{{bookend_content}}</span>
+    <span>
+        {{#if deactivated}}
+            {{t "This stream has been deactivated" }}
+        {{else}}
+            {{#if subscribed }}
+                {{#tr}} You subscribed to stream {stream} {{/tr}}
```

Comment:
Let's call the variable `stream_name`; also I think there's no reason to have the leading/trailing space here?

Reply:
Done.